dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/LimitedLeadLag.mo

```diff
@@ -0,0 +1,60 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model LimitedLeadLag "Simple lead-lag filter, with output limitation"
+
+  import Modelica;
+  import Dynawo;
+  import Dynawo.Types;
+
+  extends Modelica.Blocks.Interfaces.SISO(y(start = y_start));
+
+  parameter Types.PerUnit K = 1 "Gain";
```

Comment:
parameters coding style

Reply:
?

Reply:
For t1 and t2, YMin, YMax, etc.

Reply:
I have made, committed and pushed these changes on this block. Do you see them ?